pkg/transformers/namespace.go

```diff
@@ -97,5 +97,32 @@ func (o *namespaceTransformer) Transform(m resmap.ResMap) error {
 		}
 
 	}
+	o.updateClusterRoleBinding(m)
 	return nil
 }
+
+func (o *namespaceTransformer) updateClusterRoleBinding(m resmap.ResMap) {
+	for id := range m {
+		if id.Gvk().Kind == "ClusterRoleBinding" {
+			objMap := m[id].UnstructuredContent()
+			subjects := objMap["subjects"].([]interface{})
+			for i := range subjects {
+				subject := subjects[i].(map[string]interface{})
+				kind, foundk := subject["kind"]
+				name, foundn := subject["name"]
```

Comment:
`apiGroup` is also a supported field for subjects, so we should check against that as well.

Reply:
I will add that check

Reply:
I found this from the docs
```APIGroup holds the API group of the referenced subject. Defaults to "" for ServiceAccount subjects. Defaults to "rbac.authorization.k8s.io" for User and Group subjects.```

For `User` and `Group` subjects, we don't need to add namespace. Only `ServiceAccount` need namespace. So I think we can skip checking of `apiGroup`.